### PR TITLE
Add unnumbered support to tables and figures

### PR DESCRIPTION
**Pull request by helenemccarron**
Tables and figures with the class `.unnumbered` should not have a caption or be numbered

### CODE REVIEW

**Comment by philschatz** [comment]
Can you add an example with an unnumbered table to https://github.com/Connexions/cnx-recipes/blob/master/books/rulesets/common/examples/_all.scss#L202-L218 ? 

That way folks can see/know that it's a _special_ class that we handle properly

File: books/rulesets/common/examples/_all.scss [hunk 1]
```diff
@@ -251,6 +251,18 @@
   Style guide: book.chapter
 */
 
+/*
+  Unnumbered Content
+
+  Content that has been marked with the class .unnumbered by the content team.
+  T
```
Comment (philschatz) [inline books/rulesets/common/examples/_all.scss]: Is this a typo?

Comment (helenemccarron) [inline books/rulesets/common/examples/_all.scss]: I would appear so :-D

**Comment by philschatz** [comment]
Mind if I:

1. move this to be `page.unnumbered` (since the class isn't used on book-level elements)
2. include a Figure example

**Comment by helenemccarron** [comment]
sure, go for it

**Pull request merged**